controller/action/ContentActions.class.php

```diff
@@ -337,13 +337,19 @@ public static function prepareBioPartial(array $vars): array
   {
     $person = $vars['person'];
     $path   = 'bio/' . $person . '.md';
-    list($metadata, $bioHtml) = View::parseMarkdown($path);
-    $imgSrc         = 'https://spee.ch/@lbryteam:6/' . $person . '.jpg';
-    return $vars + $metadata + [
-      'imgSrc'      => $imgSrc,
-      'bioHtml'     => $bioHtml,
-      'orientation' => 'vertical'
-    ];
+    try{
+      list($metadata, $bioHtml) = View::parseMarkdown($path);
+      $imgSrc         = 'https://spee.ch/@lbryteam:6/' . $person . '.jpg';
+      return $vars + $metadata + [
+        'imgSrc'      => $imgSrc,
+        'bioHtml'     => $bioHtml,
+        'orientation' => 'vertical'
+      ];
+    }
+    catch(ErrorException $e){
```

Comment:
The exception is being thrown because of the missing bios file in the content section for the edge cases described below. By adding the two missing bios file, we can remove this exception completely.